src/components/Nav/Components/SubMenu.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+
+export class SubMenu extends Component {
+  render() {
+    const { sub } = this.props;
+
+    return (
+      <ul className="depth02">
+        {sub.map((value, idx) => {
+          return (
+            <li key={idx}>
+              <Link to="/">{value}</Link>
+            </li>
+          );
+        })}
```

Comment:
map 메서드 잘 활용하셨습니다! 굿굿 ! :) 
다만, 아래와 같이 매개변수 명을 좀 더 시멘틱하게 작성해주시면 더 좋을 것 같습니다. 더불어서 이동할 주소도 추후에 수정할 수 있겠네요! 
```suggestion
        {sub.map((menu, idx) => {
          return (
            <li key={idx}>
              <Link to="/">{menu}</Link>
            </li>
          );
        })}
```